components/Chips/tests/unit/ChipsTests.m

```diff
@@ -17,12 +17,29 @@
 #import <XCTest/XCTest.h>
 
 #import "../../src/MDCChipField.h"
+#import "MaterialTypography.h"
 
 // Expose internal methods for testing
 @interface MDCChipField (Testing)
 - (void)createNewChipFromInput;
 @end
 
+/** Fake MDCChipView for unit testing. */
+@interface FakeMDCChipView : MDCChipView
```

Comment:
Please name this `MDCChipsTestsFakeChipView` to avoid name conflicts. Otherwise please put it in a shared location with a header + implementation file.

Reply:
Addressed in 003cf100e75c5e196ebe217c67456116686a5c68. I think I was following the convention in FakeBottomDrawerContainerViewController to keep the name consensus across codebase.